src/cmd-generate-release-meta

```diff
@@ -146,6 +146,17 @@ def append_build(out, input_):
                     "image": cloud_dict[image_field]
                 }
 
+    # IBMCloud/PowerVS specific additions
+    for meta_key, cloud, object_field, bucket_field, url_field in ("ibmcloud", "ibmcloud", "object", "bucket", "url"), ("powervs", "powervs", "object", "bucket", "url"):
```

Comment:
optional: break this up into 3 lines since it's so long.

Reply:
done.